docs/ota-client-guide/modules/ROOT/pages/supported-boards.adoc

```diff
@@ -0,0 +1,29 @@
+== Supported boards
+
+// MC: Copied over from meta-updater on 29.07.2019
+
+Currently supported platforms are:
+
+* https://github.com/advancedtelematic/meta-updater-raspberrypi[Raspberry Pi 2 and 3]
+* https://github.com/advancedtelematic/meta-updater-minnowboard[Intel Minnowboard]
+* https://github.com/advancedtelematic/meta-updater-qemux86-64[Native QEMU emulation]
+* Renesas R-Car H3 and M3
+* https://github.com/advancedtelematic/meta-updater-ti/[TI BeagleBone Black] (rocko only, using TI SDK 05.03)
+* https://github.com/advancedtelematic/meta-updater-ti/[TI AM65x industrial development kit] (rocko only, using TI SDK 05.03)
+
+Additionally, there is community support for https://github.com/ricardosalveti/meta-updater-riscv[RISC-V] boards, in particular the Freedom U540.
+
+We also historically supported the https://github.com/advancedtelematic/meta-updater-porter[Renesas Porter] board.
+
+=== Adding support for your board
+
+If your board isn't supported yet, you can add board integration code yourself. The main purpose of this code is to provide a bootloader that will be able to use https://ostree.readthedocs.io/en/latest/manual/atomic-upgrades/[OSTree's boot directory]. In the meta-updater integration layers we have written so far, the basic steps are:
+
+1.  Make the board boot into http://www.denx.de/wiki/U-Boot[U-Boot]
+2.  Make U-boot import variables from /boot/loader/uEnv.txt and load the kernel with initramfs and kernel command-line arguments according to what is set in this file.
+
+You may take a look into https://github.com/advancedtelematic/meta-updater-minnowboard[Minnowboard] or https://github.com/advancedtelematic/meta-updater-raspberrypi[Raspberry Pi] integration layers for examples.
+
+Although we have focused on U-Boot and GRUB so far, other bootloaders can be configured to work with OSTree as well.
+
+Your images will also need network connectivity to be able to reach an actual OTA backend. Our 'poky-sota' distribution does not mandate or install a default network manager but our supported platforms use the `virtual/network-configuration` recipe, which can be used as a starting example.
```

Comment:
Can we also find a way to get rid of this copy of the text? I'd really prefer to have the list of boards in one place and have everywhere else just reference that somehow.

Reply:
See my comment on the subject here: https://github.com/advancedtelematic/aktualizr/pull/1293#discussion_r313319636